contrib/depends/packages/expat.mk

```diff
@@ -1,6 +1,6 @@
 package=expat
 $(package)_version=2.2.4
-$(package)_download_path=https://downloads.sourceforge.net/project/expat/expat/$($(package)_version)
+$(package)_download_path=https://github.com/libexpat/libexpat/releases/download/R_2_2_4
```

Comment:
Please add /monero#6500 at the end of the commit message.

Reply:
Resolved.